spec/controllers/idv/phone_confirmation_controller_spec.rb

```diff
@@ -1,192 +0,0 @@
-      allow(subject).to receive(:idv_session).and_return(idv_session)
-    end
-
-    context 'user has an existing phone number' do
-      before do
-        create(:profile, :active, :verified, user: subject.current_user, phone: '123-456-7890')
-      end
-
-      context 'user enters a valid code' do
-        before { post :confirm, code: '123' }
-
-        it 'clears session data' do
-          expect(subject.user_session[:idv_unconfirmed_phone]).to be_nil
-          expect(subject.user_session[:idv_phone_confirmation_code]).to be_nil
-        end
-
-        it 'updates user phone and phone_confirmed_at attributes' do
-          expect(subject.user_session[:idv][:params]['phone_confirmed_at']).to_not be_nil
-        end
-
-        it 'redirects to idv_questions_path' do
-          expect(response).to redirect_to(idv_questions_path)
-        end
-
-        it 'displays success flash notice' do
-          expect(flash[:success]).to eq t('notices.phone_confirmation_successful')
-        end
-      end
-
-      context 'user enters an invalid code' do
-        before { post :confirm, code: '999', otp_method: :sms }
-
-        it 'does not clear session data' do
-          expect(subject.user_session[:idv_unconfirmed_phone]).to eq('+1 (555) 555-5555')
-          expect(subject.user_session[:idv_phone_confirmation_code]).to eq('123')
-        end
-
-        it 'does not update phone_confirmed_at attribute' do
-          expect(subject.idv_session.params['phone_confirmed_at']).to be_nil
-        end
-
-        it 'redirects back phone_confirmation_path' do
-          expect(response).to redirect_to(
-            idv_phone_confirmation_path(otp_method: :sms)
-          )
-        end
-
-        it 'displays error flash notice' do
-          expect(flash[:error]).to eq t('errors.invalid_confirmation_code')
-        end
-      end
-
-      it 'tracks the confirmation event' do
-        stub_analytics
-        expect(@analytics).to receive(:track_event).
-          with('User confirmed their verified phone number')
-
-        post :confirm, code: '123'
-      end
-
-      it 'tracks an event when the user enters an invalid code' do
-        stub_analytics
-        expect(@analytics).to receive(:track_event).
-          with('User entered invalid phone confirmation code')
-
-        post :confirm, code: '999'
-      end
-    end
-  end
-
-  describe '#show' do
-    before do
-      stub_sign_in
-      subject.user_session[:idv_unconfirmed_phone] = '+1 (555) 555-5555'
-    end
-
-    it 'renders the :show template' do
-      get :show
-
-      expect(response).to render_template(:show)
-    end
-
-    context 'when trying to change asserted number' do
-      it 'sets @reenter_phone_number_path to idv sessions path' do
-        get :show
-
-        expect(assigns(:reenter_phone_number_path)).to eq(idv_session_path)
-      end
-    end
-
-    context 'when FeatureManagement.prefill_otp_codes? is true' do
```

Comment:
It's at the top of the spec: https://github.com/18F/identity-idp/blob/d95b4702b4bf8454e99ea3209796f73a9d95457c/spec/controllers/two_factor_authentication/otp_verification_controller_spec.rb#L10-L26

Since all contexts use the same view, we don't need to repeat these tests 3 times, right?